### DISCUSSION TITLE
Bad font used by cli app

### DISCUSSION THREAD

**Discussion by azzamsa** [wezterm/wezterm#4355] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

Gnome

### WezTerm version

20230712-072601-f4abf8fd

### Ask your question!

Hi.

Why `slidev` use an ugly (very thin) font in Wezterm?
How to debug it.

The left is Wezterm, The right is GNOME Terminal.

![image](https://github.com/wez/wezterm/assets/17734314/9546bbef-0d53-4edc-9852-6257cd19e163)
![image](https://github.com/wez/wezterm/assets/17734314/79fe8692-cc76-4ea3-8d1c-b2f7f41ef037)

I think it is hard to tell from the screenshot, but it is very tiny.

[Screencast from 2023-09-29 06-43-33.webm](https://github.com/wez/wezterm/assets/17734314/ee8c4a7f-495b-47f0-b70e-8dc2c98ea4f4)

Thank you for Wezterm 🏅

**Comment by wez** [comment, marked as answer]
Likely it is using something like the half intensity attribute.
Use `wezterm ls-fonts` to see which fonts will be used for different attributes.
You can use https://wezfurlong.org/wezterm/config/lua/config/font_rules.html to manage those fonts

**Reply by azzamsa** [reply]
```
    font_rules = {
        {
            intensity = "Half",
            font = wezterm.font({
                family = "Iosevka Nerd Font",
            }),
        },
```

worked like a charm! 🏅
